### ISSUE TITLE
[processor/tailsamplingprocessor] Is it possible to pass an array of numbers to policy

### ISSUE THREAD

**Issue by koungkub** [open-telemetry/opentelemetry-collector-contrib#22849]
### Component(s)

processor/tailsampling

### Describe the issue you're reporting

I'd like to pass an array of numbers on the numeric_attribute policy. for instant

```
policies:
    [
        {
            name: array_of_numbers
            type: xxx
            xxx: {key: yyy, values: [2, 4, 12, 13, 14]}
        }
    ]
```

**Comment by jiekun** [comment]
I would like to check if the OTTL policy could meet your needs later.

**Comment by koungkub** [comment]
@jiekun  It's better if we have a condition that accepted an array of numbers.

**Comment by jiekun** [comment]
> @jiekun It's better if we have a condition that accepted an array of numbers.

@koungkub I think OTTL could handle array. You could check it here: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl#lists

I'm sorry that I cannot do some test myself yesterday due to limited time. I will add some test example. And maybe you could read the OTTL doc to see if it met your expectation as well.

**Comment by koungkub** [comment]
> > @jiekun It's better if we have a condition that accepted an array of numbers.
> 
> @koungkub I think OTTL could handle array. You could check it here: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl#lists
> 
> I'm sorry that I cannot do some test myself yesterday due to limited time. I will add some test example. And maybe you could read the OTTL doc to see if it met your expectation as well.

Thanks

**Comment by jiekun** [comment]
Alright I think the boolean expression support those operation:
```
The valid operators are:

Equal (==). Tests if the left and right Values are equal (see the Comparison Rules below).
Not Equal (!=). Tests if the left and right Values are not equal.
Less Than (<). Tests if left is less than right.
Greater Than (>). Tests if left is greater than right.
Less Than or Equal To (<=). Tests if left is less than or equal to right.
Greater Than or Equal to (>=). Tests if left is greater than or equal to right.
```
The list value is acceptable for an OTTL function tho.

I would like to ping OTTL master @TylerHelmuth for help. :P

**Comment by koungkub** [comment]
@jiekun @TylerHelmuth Thank you so much. It's working correctly.

**Comment by koungkub** [comment]
@TylerHelmuth I agree with you about the precise statement of ottl. It should be accepted array of either strings or numbers

**Comment by TylerHelmuth** [comment]
@koungkub if you'd like please open a new issue for `pkg/ottl` requesting that enhancement.  Since the problem statement for this issue is solved I am going to close this issue, but ping me if you think it should be reopened.
